Junchao-Mellanox/sonic-buildimage
# [python3] Support python3 for mellanox sonic platform API and pmon related makefiles

**- Why I did it**

python2 is end of life and SONiC is going to support python3. This PR is going to support:

1. Build pmon daemons with python3
2. Install and run python3 version pmon daemons
3. Mellanox SONiC platform API python3 support

**- How I did it**

1. Change pmon daemons make files to build bothe python2 and python3 whl
2. Change docker-platform-monitor make files to install both python2 and python3 whl
3. Change pmon docker startup files to start pmon daemons according to the supported platform API version
4. Change Mellanox SONiC platform API code to python2 and python3 compatible

**- How to verify it**

1. Manual test to make sure all daemons are up and no error in syslog
2. Run platform regressions on a few platforms

**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
